src/test/resources/features/Section5/Occasional/OcccasionalJourney.feature

```diff
@@ -0,0 +1,158 @@
+    Given I fill section1 for SIMPLIFIED,prelodged declaration
+    And I fill section2
+    And I fill section3
+    And I fill section4
+
+    Then I should land on Add-Declaration-Item-1 page
+    And I select Add Item button
+    Then I should land on Procedure-codes page
+    And I select 1040 as procedure code
+    And I click continue
+    Then I should land on Additional-Procedure-Codes page
+    And I select 000 as additional procedure code
+    And I click continue
+    # skipped fiscal information , additional-fiscal-references
+    Then I should land on Commodity-Details page
+    And I enter commodity details code as 2803400090 and description as St Andrews golf ball whisky
+    And I click continue
+    And I navigate to Dangerous Goods Code page
+    And I select Yes to enter the code 1204 as UN dangerous goods
+    And I click continue
+    Then I should land on Cus-Code page
+    And I select Yes to enter the code 12345678 as CUS code
+    And I click continue
+
+    # skipped Vat Details
+    Then I should land on National-Additional-Code page
+    And I select Yes and enter A103 as national additional code
+    And I click continue
+    Then I should land on National-Additional-Codes-List page
+    And I select No for adding another national code
+    And I click continue
+
+  # Skipping Statistical-Value when additional procedure code is not a low value declaration "3LV"
+    Then I should land on Package-Information page
+    And I enter Aerosol as package type, 10 as number of packages and 1034 as shipping mark as first package info
+    And I click continue
+    Then I should land on Package-Information-List page
+    And I select No on package information list page
+    And I click continue
+    Then I should land on Additional-Information-YesNo page
+    And I select Yes to add additional information
+    And I click continue
+    Then I should land on Additional-Information page
+    And I enter 00400 as code and EXPORTER as required information as first additional information
+    And I click continue
+    Then I should land on Additional-Information-List page
+    And I select No on additional information list page
+    And I click continue
+    Then I should land on Licence-Required-YesNo page
+    And I select Yes to check if the goods require a licence
+    And I click continue
+    Then I should land on Additional-Document page
+    And I enter X002 as code and GBAEOC717572504502801 as identifier as first additional document
+    And I click continue
+    Then I should land on Additional-Document-List page
+    And I select No on additional document list page
+    And I click continue
+    Then I should land on Declaration-Items-List page
+    And I select No on declaration items list page
+    And I click continue
+    Then I should land on MiniCYA-Section-5 page
+    And I check the MiniCYA page for Section-5
+
+    # check Fiscal Reference page when Procedure code is 1042
```

Comment:
This is already tested in other journeys do we need this here?

Reply:
Yes, I know we are testing this in other journeys, i have checked skipping of the page on the top as part of journey and taught of checking if it is appearing back again in the journey. Happy to remove it if you want

Reply:
ok we will decide on the execution time later